geonode/api/api.py

```diff
@@ -156,6 +157,24 @@ class Meta:
         serializer = CountJSONSerializer()
 
 
+class OwnerResource(TypeFilteredResource):
```

Comment:
@menegon would it be possible to reuse https://github.com/GeoNode/geonode/blob/master/geonode/api/api.py#L185?